cluster/images/hyperkube/Dockerfile

```diff
@@ -40,28 +38,22 @@ RUN cp /usr/bin/nsenter /nsenter
 COPY hyperkube /hyperkube
 
 # Manifests for the docker guide
-COPY static-pods/master.json /etc/kubernetes/manifests/
-COPY static-pods/etcd.json /etc/kubernetes/manifests/
-COPY static-pods/addon-manager.json /etc/kubernetes/manifests/
-
-# TODO: Move out kube-proxy to a DaemonSet again
-COPY static-pods/kube-proxy.json /etc/kubernetes/manifests/
+COPY static-pods/master.json \
+     static-pods/etcd.json \
+     static-pods/addon-manager-singlenode.json \
+     static-pods/kube-proxy.json \
+        /etc/kubernetes/manifests/
```

Comment:
Those lines are not a problem :) The real problem is here: https://github.com/kubernetes/kubernetes/pull/30299/files#diff-86f22fb5b693949f04077054c41b762fR80
You removed all json files :)